src/test/test_verify.cpp

```
@@ -9,8 +13,61 @@
 
 using namespace prevail;
 
-#define FAIL_LOAD_ELF(dirname, filename, sectionname)                                                    \
-    TEST_CASE("Try loading nonexisting program: " dirname "/" filename, "[elf]") {                       \
+namespace {
+uint16_t read_le16(const char* data, size_t offset) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)